recipe/recipe.yaml

```
@@ -37,71 +32,95 @@ source:
         - fix-argminmax.patch
 
 build:
-  number: 0
-  skip: not (match(python, python_min ~ ".*") and is_abi3)
-  python:
-    version_independent: true
+  number: ${{ build }}
 
-requirements:
-  build:
-    - if: build_platform != target_platform
-      then:
+outputs:
+  - package:
+      name: polars
+    build:
+      script: echo "No build step is needed"
+      # Since this meta package is noarch: generic, we only need to build it
+      # for one target platform (does not matter which one)
+      skip: (target_platform != 'linux-64') or not (match(python, python_min ~ ".*") and is_abi3)
```

```suggestion
      skip: target_platform != 'linux-64'
```

i think this should be fine as well